website/docs/utilities/dialog-primitive/partials/code/component-api.md

```diff
@@ -0,0 +1,91 @@
+    A named block where the content for the “body” or main content of the dialog is rendered. The `DialogPrimitive::Body` component should be added here.
+  </C.Property>
+  <C.Property @name="<:footer>" @type="named block">
+    A named block where the content for the “footer” section of the dialog is rendered. The `DialogPrimitive::Footer` component should be added here.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### DialogPrimitive::Header
+
+A sub-component used to provide the header content.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="icon" @type="string">
+    Accepts any [icon](/icons/library) name.
+  </C.Property>
+  <C.Property @name="tagline" @type="string">
+    Text above the title that can be used to provide more context.
+  </C.Property>
+  <C.Property @name="yield">
+    Content passed as children of this sub-component is yielded to the “title” block of the header.
+  </C.Property>
+   <C.Property @name="onDismiss" @type="function">
+    Callback function invoked when the “dismiss” button in the header is clicked by the user.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### DialogPrimitive::Description
+
+A sub-component used to provide an extra description to the header.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="yield">
+    Elements passed as children of this sub-component are yielded inside the “description” block of the header. It comes with a pre-defined typographic style.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### DialogPrimitive::Body
+
+A sub-component used to contain the main content of the dialog. When the yielded content exceeds the available space, a scrollbar is introduced to the container.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="yield">
+    Elements passed as children of this sub-component are yielded inside the element.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### DialogPrimitive::Footer
+
+A sub-component used to provide the footer’s content. We recommend using it exclusively for actions using the [ButtonSet](/components/button-set) component. If a tertiary action is presented, it will always be aligned at the end of the row.
```

Comment:
```suggestion
A sub-component used to provide the footer’s content. We recommend using it exclusively for actions using the [ButtonSet](/components/button-set) component. If a tertiary action is included, it will always be aligned at the end of the row.
```